src/common/db/schemas.lua

```diff
@@ -4,16 +4,19 @@ schemas.ant_index = [[
         CREATE TABLE IF NOT EXISTS ant_index (
             ant_id TEXT PRIMARY KEY NOT NULL, 
             owner TEXT NOT NULL,
-            registered_at INTEGER NOT NULL
+            registered_at INTEGER NOT NULL,
+            last_updated INTEGER NOT NULL
```

Comment:
wonder if aos-sqlite supports triggers? https://www.w3resource.com/sqlite/sqlite-triggers.php

this would be opposed to providing the timestamps manually

Reply:
checking, heres the lua-sqlite docs: http://lua.sqlite.org/index.cgi/doc/tip/doc/lsqlite3.wiki#db_errmsg

Reply:
Looks like we can use triggers, there was mentions of them in the docs though no examples of creating them... however the timestamp will not be accurate I think, since we need to use msg.Timestamp - os.time() is actually available, and it gets that from the SU, but im not entirely sure if thats passed into the sqlite-context appropriately